tumblr/kanvas-ios
# Videos have the wrong size when exporting them with a filter

After applying a filter to a video, when exporting it the dimensions were wrong.
This PR removes the size specification in the renderer used by the `VideoCompositor` so that the default is used.

To test it in Orangina:
-Open the editor to create a new post.
-Select a video from your phone or record a new one.
-Add a filter to your video and tap on the blue arrow to select it.
-The size of the video in the editor shouldn't look distorted or different.
-Post the video.
-The video should be posted with the selected filter and the correct dimensions.
-Bonus: try with videos in different orientations and with images and GIFs too, to make sure they weren't affected.
